Guided Hunting - Azure Resource Explorer.ipynb

```diff
@@ -0,0 +1,7311 @@
+{
```

Comment:
<p>query providers have their own query_time objects now.</p><p>You could use qry_prov,query_time as an alternative.</p><p>No need to change the code you have here though - just FYI</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Azure/Azure-Sentinel-Notebooks/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Got it, will keep this in mind in the next notebook! Thanks!</p>